ui/views.py

```diff
@@ -280,3 +281,33 @@ def serve_media(request, media_path):
     response['Content-Length'] = os.path.getsize(file_path)
     response['Content-Disposition'] = "attachment; filename=%s" % filename
     return response
+
+
+@login_required
```

Comment:
It matches the filename to the user inside the function to restrict users downloading others exports